utils/timer_delay.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * timer_delay.h
+ *
+ *  Created on: 19-Feb-2021
+ *      Author: Sonal Shrivastava
+ */
```

Comment:
Can you please add the GPL header to the top of the file (copy over from other files)?

Reply:
yes, done